x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Nice! Also It will be nice to extend that test in the future 🚀 ( I have some ideas about what else we can cover )